### PR TITLE
Enzyme support older versions

### PR DESCRIPTION
**Pull request by wsmoses**

### CODE REVIEW

File: ext/Enzyme012Ext.jl [hunk 1]
```diff
@@ -0,0 +1,342 @@
+# https://github.com/EnzymeAD/Enzyme.jl/issues/1516
```
Comment (vchuravy) [inline ext/Enzyme012Ext.jl:1]: Can we name these files after the EnzymeCore version?

Comment (wsmoses) [reply ext/Enzyme012Ext.jl]: done

**Comment by wsmoses** [comment]
is there a similar clang-format like command I can run to auto fix?

**Comment by vchuravy** [comment]
Yeah you can https://github.com/fredrikekre/Runic.jl

```
julia --project=@runic -e 'using Pkg; Pkg.add(url = "https://github.com/fredrikekre/Runic.jl")'
alias runic="julia --project=@runic -e 'using Runic; exit(Runic.main(ARGS))' --"
 runic -i ext src
```

**Pull request merged**